spec/helpers/retina_image_helper_spec.rb

```diff
@@ -5,7 +5,7 @@
 
   before { allow(helper).to receive(:image_path) { |file| "/images/#{file}" } }
 
-  it { assert_match(%r{\A<img src=".*" srcset=".*" alt=".*" />\z}, tag) }
+  it { assert_match(%r{\A<img src=".*" srcset=".*" alt=".*"\s*\/?>}, tag) }
```

Comment:
I'm not sure why but the Rails `tag` helper seems to sometimes generate ` />`, ` >`, `>` at different times. This isn't the first time we have had to adjust this spec but hopefully we have all the cases now.